### DISCUSSION TITLE
Bad data in Mugello

### DISCUSSION THREAD

**Discussion by McHouse** [theOehrly/Fast-F1#562] [Q&A]
Hello everyone
I think the 2020 TUSCAN GP circuit data (number 9) in Mugello are confused.
I say this, because that track is 5245m. with 15 turns, and in circuit_info.corners it appears with less than 4500m and 27 turns.

There's no rush (theOehrly) :giggles:

Thanks in advance.

**Comment by theOehrly** [comment]
I created issue #563 for this. Thanks for reporting
